android/app/src/main/java/com/reactnativeapp/MainApplication.java

```diff
@@ -38,6 +39,7 @@ public boolean getUseDeveloperSupport() {
         protected List<ReactPackage> getPackages() {
             return Arrays.asList(
                     new MainReactPackage(),
+            new RNGestureHandlerPackage(),
```

Comment:
please align this with the rest, my OCD is ticking

Reply:
Haha I will, but that is `react-native link` doing it :)

Reply:
I know, but I always go in and add these extra tabs to this file 🙈